lms/static/js/spec/staff_debug_actions_spec.js

```diff
@@ -41,9 +49,45 @@ define(['backbone', 'jquery', 'js/staff_debug_actions'],
                 it('gets the placeholder name if the id has escapable characters', function() {
                     $('body').append(escapableFixture);
                     expect(StaffDebug.get_user('test.*+?^:${}()|][loc')).toBe('userman');
-                    $('#' + escapableFixture_id).remove();
+                    $("input[id^='sd_fu_']").remove();
                 });
             });
+            describe('do_idash_action success', function () {
+                it('adds a success message to the results DOM element', function () {
+                    $('body').append(escapableResultArea);
+                    var requests = AjaxHelpers.requests(this);
+                    var action = {
+                        locationName: esclocationName,
+                        location: location,
+                        method: 'reset_student_attempts',
+                        success_msg: gettext('Successfully reset the attempts for user userman'),
```

Comment:
Don't call gettext in tests-- it forces things to be translated that may not actually be present in the production (causing unnecessary translation).